test/unit/StakingPool/calculatePremium.js

```diff
@@ -0,0 +1,352 @@
+const { expect } = require('chai');
+const { ethers } = require('hardhat');
+const { parseEther } = ethers.utils;
+const { BigNumber } = ethers;
+const { calculateSurgePremiums, calculateBasePrice, divCeil } = require('./helpers');
+const { daysToSeconds } = require('../../../lib/helpers');
+
+describe('calculatePremium', function () {
```

Comment:
i could be missing it but i can't see the test case for:

`initialCapacityUsed < surgeStartPoint & finalCapacityUsed < surgeStartPoint`